.github/workflows/link_and_spell_check.yml

```diff
@@ -0,0 +1,48 @@
+on:
+  workflow_dispatch:
+  pull_request:
+    branches:
+    - main
+
+name: "Check spelling, linting and links"
+
+jobs:
+  check-markdown:
+    runs-on: ubuntu-latest
+    name: "Check spelling, linting and links"
+    steps:
+    - uses: actions/checkout@v2
+    - run: |
+        wget https://github.com/qir-alliance/.github/archive/refs/heads/main.zip -O clone.zip &&
+        unzip clone.zip && mv .github-main out && rm clone.zip &&
+        mv out/workflow-templates/utils/ utils && rm -r out
```

Comment:
It would be great with some comments about what is happening here.

Reply:
I added a comment.